CanHazFunny/CanHazFunny/JokeModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿
+namespace CanHazFunny
+{
+    public class JokeModel
+    {
+        // Geek jokes API model uses lowercase in JSON
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Naming", "INTL0002:Properties PascalCase", Justification = "<Pending>")]
+        public string joke { get; set; } = string.Empty;
```

Comment:
```suggestion
        [JsonProperty("joke")]
        public string Joke { get; set; } = string.Empty;
```

You could use the `JsonProperty` decorator to specify the json name for this property. (You might need to import `Newtonsoft.json` to use this)